### PR TITLE
Update coursier version, support arm64

### PR DESCRIPTION
**Pull request by etspaceman**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -105,7 +106,9 @@
     "concat-map": "^0.0.2",
     "follow-redirects": "^1.14.9",
     "semver": "^7.5.0",
-    "vscode-languageclient": "^8.0.0"
+    "unzip-stream": "^0.3.4",
```
Comment (kubukoz) [inline package.json:111]: **issue:** two entries for this dep, apparently - let's delete one

File: src/coursier/coursier.ts [hunk 2]
```diff
@@ -9,7 +9,10 @@ export function getCoursierExecutable(extensionPath: string): Promise<string> {
       return downloadCoursierIfRequired(
         extensionPath,
         process.platform,
-        "v2.0.13"
+        process.arch,
+        // Have to use a commit hash for now since the launchers repository is untagged
+        "56971135cd9b08eaefed13b4d6b7a95ba9cce572",
```
Comment (kubukoz) [inline src/coursier/coursier.ts:14]: **question:** did you reach out to Alex & co. to see if they can tag these commits in the future?

Comment (etspaceman) [reply src/coursier/coursier.ts:14]: I'll reach out in the coursier repository as an issue.

Comment (etspaceman) [reply src/coursier/coursier.ts:14]: https://github.com/coursier/coursier/issues/2998

Comment (kubukoz) [reply src/coursier/coursier.ts:14]: thankss

Comment (kubukoz) [reply src/coursier/coursier.ts:14]: actually, would you mind putting that link in the code comment?

Comment (etspaceman) [reply src/coursier/coursier.ts:15]: Done

**Review by kubukoz — APPROVED** [review]

**Review by daddykotex — APPROVED** [review]
Thanks @etspaceman 

I can't confirm everything is working as I did not have the time to try it out locally, but the code LGTM

Thanks for the additional test & docs

**Pull request merged**